lib/rebay_api_web/controllers/bid_controller.ex

```diff
@@ -32,13 +32,45 @@ defmodule RebayApiWeb.BidController do
     |> Map.put("uuid", Ecto.UUID.generate())
 
     with {:ok, %Bid{} = bid} <- UserItem.create_bid(bid_params) do
+      handle_auto_bids(item.id)
+
       conn
       |> put_status(:created)
       |> put_resp_header("location", Routes.item_bid_path(conn, :show, item.uuid, bid))
       |> render("show.json", bid: bid)
     end
   end
 
+  defp handle_auto_bids(item_id) do
+    item_id
+    |> UserItem.get_auto_bids_by_item
+    |> run_auto_bids(item_id)
+
+    case length(item_id |> UserItem.get_auto_bids_by_item) do
```

Comment:
Could you reduce duplication and improve readability by assigning the collection of bids to a variable?